egs2/TEMPLATE/enh1/enh.sh

```diff
@@ -79,7 +79,7 @@ inference_model=valid.loss.ave.pth
 download_model=
 
 # Evaluation related
-scoring_protocol="STOI SDR SAR SIR SI_SNR"
+scoring_protocol="STOI SDR SAR SIR SI_SNR PESQ"
```

Comment:
We'd better keep the previous value for now, as the license of the PESQ program is not friendly to open-source projects.